### DISCUSSION TITLE
non-logged in users getting blank white page in Chrome

### DISCUSSION THREAD

**Discussion by whiteatom** [requarks/wiki#4720] [Help / Questions] [answered]
I'm in a mess here and I can't figure out the problem. Relatively new Wiki.js install and all was going great. 

I can login using Safari and BigSur and all is well. However, when I try to access the Wiki from Chrome, I just get a blank white screen. 

I tried enabling an OAuth authorization system - that's working, and if I login to the master site (Wordpress), and then browse the wiki, it works! However, if I try to access the wiki directly while not logged in, I get the blank white screen.

Some digging has reviled an error on the JS console: Failed to load resource: net::ERR_INCOMPLETE_CHUNKED_ENCODING for app.js:1. I have tried clearing my cache, hard refreshes and even rebooting/clearing and it still gives the JS error loading the app.js.

I don't know if this was a problem on a clean install because I only found out about it when someone else tried to log in.

Is this some weird HTTP header issue? Can anyone help me troubleshoot this?

**Comment by Phil-hostwiki** [comment, marked as answer]
You may want to checkout the answers to this Stackoverflow question similar to your situation.   
https://stackoverflow.com/questions/29894154/chrome-neterr-incomplete-chunked-encoding-error

**Reply by whiteatom** [reply]
Thanks - I did see that one, but the solution was a virus scanner. I'm on a Mac, no virus scanner, and this is 100% reproducible on multiple chrome clients.

**Reply by whiteatom** [reply]
Ok.. did some more digging.. turns out the root fs on the server was full. Deleted some old logs and all is well again. Strange that this was a Chrome-only issue...
